### PR TITLE
DFAST-226: Resolve criticalsecurity issue when trying to open user manual

### PR DESCRIPTION
**Pull request by ralph-peelen-deltares**
Using default pdf viewer and disable locking of view when pdf is open.

Explanation of File Descriptors
File descriptors are integer handles that represent open files or other I/O resources, such as sockets and pipes. They are used by the operating system to manage these resources. When a new process is spawned, it can inherit file descriptors from the parent process.

Why Use **close_fds**=True
Avoid Blocking: If a file descriptor is tied to a network socket or a pipe, keeping it open in the child process could cause the process to hang or block, especially if the parent process is waiting for data from or sending data to that file descriptor.

### CODE REVIEW

**Review by hrajagers — CHANGES_REQUESTED** [review]
You have seen the failing test?
Otherwise just a few small changes.

File: dfastmi/gui/dialog_utils.py
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_utils.py:255]: will do

File: dfastmi/gui/dialog_utils.py
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_utils.py:263]: i'll wait for an approval of @MRVermeulenDeltares, but will merge with main. If green we will revert this

File: dfastmi/gui/dialog_utils.py
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_utils.py:264]: is this double except needed if you use the same exception message twice? Or do you raise the error to a higher lvl?

File: dfastmi/gui/dialog_utils.py
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_utils.py]: implemented

File: dfastmi/gui/dialog_utils.py
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_utils.py]: implemented

File: dfastmi/gui/dialog_utils.py
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_utils.py]: @MRVermeulenDeltares is right, implemented

File: dfastmi/gui/dialog_utils.py
Comment (MRVermeulenDeltares) [inline dfastmi/gui/dialog_utils.py:271]: Private method
```suggestion
def _get_default_pdf_reader_windows():
```

Also update the method call

Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_utils.py:271]: yes good idea

File: tests/gui/test_dialog_view.py
Comment (MRVermeulenDeltares) [reply tests/gui/test_dialog_view.py:370]: Discussed, test currently not needed.
Previous test was too strict.

File: dfastmi/gui/dialog_utils.py
Comment (ralph-peelen-deltares) [reply dfastmi/gui/dialog_utils.py:264]: we discussed this and leave as is

Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_utils.py:264]: Discussed, this is done for a message in the qt view.

**Review by MRVermeulenDeltares — APPROVED** [review]

File: dfastmi/gui/dialog_utils.py
Comment (MRVermeulenDeltares) [reply dfastmi/gui/dialog_utils.py:263]: approved.

**Review by hrajagers — APPROVED** [review]

**Pull request merged**
